src/client/pfs/pfs.proto

```diff
@@ -179,6 +179,19 @@ message ObjectInfo {
   BlockRef block_ref = 2;
 }
 
+message Merge {
+  repeated string prefixes = 1;
+}
```

Comment:
So for my understanding, is a `Merge` an example of something that would be put in the `Data` field of a `Task`?

Reply:
Yes.